feed-integration-webapp/src/main/java/org/bahmni/module/feedintegration/crater/CraterAPIcalls.java

```diff
@@ -0,0 +1,158 @@
+package org.bahmni.module.feedintegration.crater;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPatientFullRepresentation;
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPatientIdentifier;
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPersonName;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Optional;
+
+import static com.sun.org.apache.xml.internal.utils.LocaleUtility.EMPTY_STRING;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class CraterAPIcalls {
```

Comment:
Instead of calling this class `CraterAPIcalls` can we rename it to something like: `CraterAPIClient` or `CraterWrapper` or just `Crater`?

Reply:
Name changed to CraterAPIClient.